mozilla-release/modules/libpref/init/all.js

```diff
@@ -4047,16 +4048,16 @@ pref("network.psl.onUpdate_notify", false);
 pref("geo.provider.network.timeout", 60000);
 
 #ifdef XP_MACOSX
-  pref("geo.provider.use_corelocation", true);
+  pref("geo.provider.use_corelocation", false);
```

Comment:
@aleksei-cliqz we had this true in `1.33`, are you sure to make this false?

Reply:
> @aleksei-cliqz we had this true in `1.33`, are you sure to make this false?

That is what we had in 1.33.1
`#ifdef XP_MACOSX
  pref("geo.provider.use_corelocation", false);
#endif

// Set to false if things are really broken.
#ifdef XP_WIN
  pref("geo.provider.ms-windows-location", false);
#endif

#if defined(MOZ_WIDGET_GTK) && defined(MOZ_GPSD)
  pref("geo.provider.use_gpsd", false);
#endif`

Reply:
df42477b28a72116811c112b684966b6e4b707ce

Reply:
My bad, I got confused with something else, apologies.

Reply:
> My bad, I got confused with something else, apologies.

No worries, that is why we work together as a team man;)